css/main.css

```diff
@@ -118,3 +118,92 @@ header .container button {
     height: 30px;
     width: 177px;
 }
+
+/*footer =======================================================*/
+
+footer {
+    background: black;
+    color: white;
+}
+
+footer .inner {
+    display: flex;
+    justify-content: center;
+    padding: 100px 0 54px;
+}
+
+footer .inner .inner-information {
+    max-width: 224px;
+}
+
+footer .inner .inner-information .inner-information-title {
+    font-size: 24px;
+    font-weight: 800;
+}
+
+footer .inner .inner-information .inner-information-text {
+    font: 13px/22px "Inter";
```

Comment:
I see that you duplicate the font style "Inter" every time, can we use this in global to not duplicate this one?